### DISCUSSION TITLE
EL3 MyOpenMath

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#377] [General]
Reviewer: @tdegeorge 

1-5 - convert exp/log
6-13 - evaluate logs with and without calculator
14 - domain

**Comment by tdegeorge** [comment]
The only question I might be concerned about is problem 10. None of the examples in the activity book has ones like it:
<img width="530" alt="Screenshot 2024-07-04 at 1 01 52 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/bc0397b5-86bd-4599-bd58-0945f12cfd36">
